feature-libs/product-configurator/rulebased/core/model/configurator.model.ts

```diff
@@ -143,19 +154,22 @@ export namespace Configurator {
   }
 
   export enum UiType {
-    NOT_IMPLEMENTED = 'not_implemented',
```

Comment:
according to this `changing the order for exported enums`, it is breaking change.

Reply:
picked

Reply:
Done